scripts/envs/audio-to-text

```diff
@@ -12,8 +12,8 @@ export APP_RUN_COMMAND="streamlit run whisper_client.py"
 export INIT_CONTAINER="quay.io/redhat-ai-dev/whisper-small:latest"
 
 # https://github.com/containers/ai-lab-recipes/blob/main/model_servers/whispercpp/base/Containerfile
```

Comment:
I think we should update this link to this https://github.com/redhat-ai-dev/developer-images/blob/main/model-servers/whispercpp/1.7.6/Containerfile